type_data/migrations/33_osa_npcs.rb

```diff
@@ -0,0 +1,104 @@
+migrate :aggressive_npc do
+  insert(:name, %{battle\-scarred hulking krolvin warden})
+  insert(:name, %{battle\-scarred krolvin triggerman})
+  insert(:name, %{battle\-scarred krolvin warden})
+  insert(:name, %{dwarven pillager})
+  insert(:name, %{dwarven pirate})
+  insert(:name, %{dwarven raider})
+  insert(:name, %{dwarven scout})
+  insert(:name, %{elven pillager})
+  insert(:name, %{elven pirate})
+  insert(:name, %{elven raider})
+  insert(:name, %{elven scout})
+  insert(:name, %{elven waylayer})
+  insert(:name, %{erithian pillager})
+  insert(:name, %{erithian pirate})
+  insert(:name, %{erithian raider})
+  insert(:name, %{erithian waylayer})
+  insert(:name, %{giantman pillager})
+  insert(:name, %{giantman pirate})
+  insert(:name, %{giantman raider})
+  insert(:name, %{giantman scout})
+  insert(:name, %{giantman waylayer})
+  insert(:name, %{gnomish pillager})
+  insert(:name, %{gnomish pirate})
+  insert(:name, %{gnomish raider})
+  insert(:name, %{gnomish scout})
+  insert(:name, %{gnomish waylayer})
+  insert(:name, %{grizzled hulking krolvin huntmaster})
+  insert(:name, %{grizzled hulking krolvin triggerman})
+  insert(:name, %{grizzled hulking krolvin warden})
+  insert(:name, %{grizzled krolvin huntmaster})
+  insert(:name, %{grizzled krolvin warden})
+  insert(:name, %{half\-elven pillager})
+  insert(:name, %{half\-elven pirate})
+  insert(:name, %{half\-elven raider})
+  insert(:name, %{half\-elven scout})
```

Comment:
might make sense to have tests for these, seems weird these aren't already escaped when the regexp is compiled

Reply:
It might already do that, I just escaped them by default. Let me modify the code and see what happens with the built in test as other .rb migration files don't escape the characters either.